modules/ignitions/eks-kube-config/resources/kubernetes/kubeconfig

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+preferences: {}
+kind: Config
+
+clusters:
+- cluster:
+    server: ${cluster_endpoint}
+    certificate-authority-data: ${cluster_auth_base64}
+  name: ${kubeconfig_name}
+
+contexts:
+- context:
+    cluster: ${kubeconfig_name}
```

Comment:
Of course, it does not matter to use the same name, but I guess could consider to use ${cluster_name} here.